apps/minifront/src/state/ibc-out.ts

```diff
@@ -215,20 +216,44 @@ const getPlanRequest = async ({
 
   const { timeoutHeight, timeoutTime } = await getTimeout(chain.channelId);
 
+  // Request transparent address from view service
+  const { address: t_addr, encoding: _encoding } = await penumbra
+    .service(ViewService)
+    .transparentAddress(new TransparentAddressRequest({}));
+  if (!t_addr) {
+    throw new Error('Error with generating IBC transparent address');
+  }
+
+  // IBC-related fields
+  let denom = getMetadata(selection.balanceView).base;
+  let useTransparentAddress = false;
+
+  // Temporary: detect USDC Noble withdrawals, and use transparent (t-addr) addresses
+  // to ensure bech32m encoding compatibility.
+
+  if (denom.includes('uusdc') && chain.chainId == 'noble-1') {
+    // Use the t-addr for USDC withdrawals, and don't override existing
+    // 'useCompatAddress' field to maintain backwards compatibility.
+    useTransparentAddress = true;
+    // Set the return address to the t-addr.
+    returnAddress = t_addr;
+  }
+
   return new TransactionPlannerRequest({
     ics20Withdrawals: [
       {
         amount: toBaseUnit(
           BigNumber(amount),
           getDisplayDenomExponentFromValueView(selection.balanceView),
         ),
-        denom: { denom: getMetadata(selection.balanceView).base },
+        denom: { denom },
         destinationChainAddress,
         returnAddress,
         timeoutHeight,
         timeoutTime,
         sourceChannel: chain.channelId,
         useCompatAddress: bech32ChainIds.includes(chain.chainId),
```

Comment:
This field is deprecated, so it can be omitted entirely and set to false

Reply:
if we want to be explicit about maintaining backwards compatibility with older frontends, wouldn't we want to keep this field?

Reply:
removed the deprecated field